File: lua/refactoring/tests/debug/printf/go/simple-function/printf.expected.go
package main

import "fmt"

func simple_function(a int) {
    var test int = 1
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
        // __AUTO_GENERATED_PRINTF_START__
        fmt.Println("simple_function 1") // __AUTO_GENERATED_PRINTF_END__
    }
}

File: lua/refactoring/tests/debug/printf/cs/class-function/printf.expected.cs
class Extract
{
    public static void simpleFunction(int a)
    {
        int test = 1,
            test_other = 11;
        for (int x = 0; x < test_other + test; x++)
        {
            Console.WriteLine(x + " " + a);
            // __AUTO_GENERATED_PRINTF_START__
            Console.WriteLine(@"Extract#simpleFunction#for 1"); // __AUTO_GENERATED_PRINTF_END__
        }
    }
}
